pkg/controller/releasemanager/state.go

```diff
@@ -132,7 +181,7 @@ func Deployed(ctx context.Context, deployment Deployment) (State, error) {
 	if !deployment.isDeployed() {
 		return deploying, nil
 	}
-	if deployment.isTestPending() {
+	if deployment.getExternalTestStatus().Enabled() {
```

Comment:
I'm not sure if this is the same behavior, but I think the intention before was to only test the first time the revision transitions out of deployed. I would think this would test every time a revisions is transitions out of deployed. That may be what we want, but it will slow down rollbacks to retired versions. Not saying we need to change, just define I guess?

Reply:
oooh good point